index.html

```diff
@@ -128,13 +128,16 @@ <h2>contact</h2>
         <textarea id="topic"></textarea>
       </form>
       <img src="assets/office.jpg" alt="Hello agency headquaters">
-      <p>
+      <div>
         <span>Our office is located in the heart of Warsaw</span>
-        <span>Feel free to say hello!</span>
-        <span>Warsaw St. 23/21</span>
-        <span>03-323 Warsaw</span>
-        <span>POLAND</span>
-      </p>
+        <span>Feel free to say hello!</span>>
+        <div itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+          <span itemprop="streetAddress">Warsaw St. 23/21</span>
+          <span itemprop="postalCode">03-323</span>
+          <span itemprop="addressLocality">Warsaw</span>
+          <span itemprop="addressCountry">POLAND</span>
```

Comment:
❤️ Just this capslock left here... 😄

Reply:
I just pushed that, wanted a separate commit 😛